packages/frontend/src/components/common/modal/ChartCreateModal.tsx

```diff
@@ -106,6 +126,36 @@ const ChartCreateModal: FC<ChartCreateModalProps> = ({
         showSpaceInput,
     ]);
 
+    const handleSaveChartInDashboard = async () => {
+        if (
+            fromDashboard === null ||
+            dashboardUuid.length === 0 ||
+            !selectedDashboard
+        )
+            return;
+        await updateDashboard({
+            name: fromDashboard,
+            filters: selectedDashboard?.filters,
+            tiles: [
+                ...selectedDashboard.tiles,
+                {
+                    type: DashboardTileTypes.SAVED_CHART,
+                    properties: {
+                        savedChartUuid: uuid4(),
+                        newChartData: {
+                            ...savedData,
+                            name,
+                            description,
+                        },
+                    },
+                    ...getDefaultChartTileSize(savedData.chartConfig?.type),
+                },
+            ],
+        });
+        history.push(`/projects/${projectUuid}/dashboards/${dashboardUuid}`);
+        sessionStorage.clear();
```

Comment:
If I edit the URL, the session storage isn't cleared. 
<img width="1610" alt="Screenshot 2023-07-20 at 15 58 57" src="https://github.com/lightdash/lightdash/assets/9117144/11912de0-ae4d-4bbe-a2a9-1095c0522df3">